cmd/config/config.go

```diff
@@ -28,6 +28,9 @@ func NewConfigCmd(analyticsTracker analytics.Tracker) *cobra.Command {
 		Short: "View and modify specific configuration values",
 		Use:   "config",
 		PreRun: func(cmd *cobra.Command, args []string) {
+			if viper.GetBool(cliflags.AnalyticsOptOut) {
```

Comment:
I feel like we could pass this value to the `Send*` methods (since there are going to be a couple after [this PR](https://github.com/launchdarkly/ldcli/pull/200) is merged) and let the analytics tracker do a no-op if it's true. Then we wouldn't need to check every place the `Send*` methods are called.

That said, if it's easier to merge this and then do another PR once the above PR is merged, then go for it.

Reply:
easy enough! updated